Backend-Development/README.md

```diff
@@ -12,25 +12,25 @@
 - [HTML](https://www.youtube.com/watch?v=kUMe1FH4CHE)
 
 ### Build a project
-- [Resourse 1](https://www.frontendpractice.com/)
-- [Resourse 2](https://www.frontendmentor.io/)
+- [Resource 1 : frontendpractice.com](https://www.frontendpractice.com/)
```

Comment:
Remove the `Resource 1`, `Resource 2` tags, and just add the resource name.
For example:
Before: `Resource 1: Javascript Course `
After: `Javascript Course`

Reply:
on it sir!